sympy/simplify/simplify.py

```diff
@@ -1156,57 +1157,6 @@ def f(rv):
     return bottom_up(expr, f)
 
 
-def walk(e, *target):
```

Comment:
Maybe add this back as a function with a deprecation warning that calls the other walk function from core.

Reply:
Will do that for all removed functions.